en/docs/getting-started/about-this-release.md

```diff
@@ -2,113 +2,108 @@
 
 WSO2 API Manager is a platform for creating, managing, consuming, and monitoring APIs. It employs proven SOA best practices to solve a wide range of API management challenges such as API provisioning, API governance, API security, and API monitoring. It combines some of the most powerful and mature components of the WSO2's state-of-the-art Carbon platform to deliver a smooth and end-to-end API management experience while catering to both API publisher and API consumer requirements.
 
-WSO2 API Manager comprises of the following modules:
+WSO2 API Manager comprises of the following modules:
 
--   **Publisher Portal**: Define new APIs and manage them.
--   **Developer Portal**: Browse published APIs and subscribe to them.
--   **API Gateway**: The underlying API runtime based on [WSO2 Enterprise Integrator (WSO2 EI) 6.6.0](https://docs.wso2.com/display/EI660)
--   **API Key Manager**: Performs key generation and key validation functionalities.
--   **API Traffic Manager**: Performs rate limiting of API requests.
+* **Publisher Portal**: Define new APIs and manage them.
+* **Developer Portal**: Browse published APIs and subscribe to them.
+* **API Gateway**: The underlying API runtime based on [WSO2 Enterprise Integrator (WSO2 EI) 6.6.0](https://docs.wso2.com/display/EI660)
+* **API Key Manager**: Performs key generation and key validation functionalities.
+* **API Traffic Manager**: Performs rate limiting of API requests.
```

Comment:
we can use any of these syntax. Any is fine. https://daringfireball.net/projects/markdown/syntax#list

Reply:
Will revert the changes back to `-` :). It is good to maintain the same style for listing across the docs.